apps/console/src/features/applications/components/forms/inbound-saml-form.tsx

```diff
@@ -637,34 +640,37 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
-                                    }
-                                    name="signatureValidationCertAlias"
-                                    type="dropdown"
-                                    required={ false }
-                                    disabled={ !isRequestSignatureValidationEnabled }
-                                    value={ initialValues?.requestValidation.signatureValidationCertAlias }
-                                    requiredErrorMessage={
-                                        t("console:develop.features.applications.forms.inboundSAML.sections" +
-                                            ".requestValidation.fields.signatureValidationCertAlias.validations.empty")
-                                    }
-                                    default={ metadata?.certificateAlias.defaultValue }
-                                    children={ getAllowedOptions(metadata?.certificateAlias) }
-                                    readOnly={ readOnly }
-                                    data-testid={ `${ testId }-request-validation-certificate-alias-dropdown` }
-                                />
-                                <Hint disabled={ !isRequestSignatureValidationEnabled }>
-                                    { t("console:develop.features.applications.forms.inboundSAML.sections" +
-                                        ".requestValidation.fields.signatureValidationCertAlias.hint") }
-                                </Hint>
-                            </Grid.Column>
-                        </Grid.Row>
+                        {
+                            isSignatureValidationCertificateAliasEnabled &&
+                            <Grid.Row columns={ 1 }>
+                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 8 }>
+                                    <Field
+                                        ref={ signatureValidationCertAlias }
+                                        label={
+                                            t("console:develop.features.applications.forms.inboundSAML.sections" +
+                                                ".requestValidation.fields.signatureValidationCertAlias.label")
+                                        }
+                                        name="signatureValidationCertAlias"
+                                        type="dropdown"
+                                        required={ false }
+                                        disabled={ !isRequestSignatureValidationEnabled }
+                                        value={ initialValues?.requestValidation.signatureValidationCertAlias }
+                                        requiredErrorMessage={
+                                            t("console:develop.features.applications.forms.inboundSAML.sections" +
+                                                ".requestValidation.fields.signatureValidationCertAlias.validations.empty")
```

Comment:
Possible character count limit exceeding. Please check and limit line character limit to 120 max.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1526/commits/69aece1b3e9eb4a72d9cf5496e593dfde17523f9.